website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/developer-reference.mdx

```diff
@@ -49,20 +49,28 @@ Server Development > Script Include.
 | Class Name            | Description                                                |
 | --------------------- | ---------------------------------------------------------- |
 | `tf_config`           | Helper to pull values from the SN Terraform Configs Table  |
+| `tf_get_workspace`    | Client-callable script to retrieve workspace data          |
 | `tf_http`             | ServiceNow HTTP REST Wrapper for requests to Terraform API |
 | `tf_run`              | Resources for Terraform Run API Requests                   |
 | `tf_terraform_record` | Manage ServiceNow Terraform Table Records                  |
+| `tf_test_config`      | Client-callable script to test Teraform connectivity       |
 | `tf_util`             | Miscellaneous helper functions                             |
 | `tf_variable`         | Resources for Terraform Variable API Requests              |
 | `tf_vcs_record`       | Manage ServiceNow Terraform VCS Repositories Table Records |
-| `tf_workspace`        | Resources for Terraform Workspace API Requests             |
+| `tf_workspace`        | Resources for Terraform workspace API Requests             |
 
 ### Example Service Catalog Flows and Actions
 
-The ServiceNow Example Flows can be found in the ServiceNow Studio > Flows. By
-default, the flows execute upon submitting an order request for the various
-catalog items. Admins can modify the flows and Actions to wait on an approval
-action, include approval rules, and specify approver groups.
+-> The ServiceNow Service Catalog for Terraform provides sample Catalog Items that utilize both **Flows** 
+and **Workflows** as their primary process engines. To determine a specific engine used by an Item, open it 
+in the edit mode and navigate to the **Proces Engine** tab. For instance, **Create Workspace** uses a **Workflow**, 
+whereas **Create Workspace Flow** is built upon a **Flow**. Both can be accessed in the **Studio**. You can also 
+manage **Flows** in the **Flow Designer**. For **Workflows**, navigate to **All > Workflow Editor**. **Flows** are 
+a newer solution developed by ServiceNow and are generally preferred over **Workflows**.
```

Comment:
The final sentence about flows being the perferred solution feels like good content for the beginning of this paragraph. Maybe we should add it to the top of this section?